src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests.Android/Resources/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Might be worth adding Resizetizer here to reduce these files and also it is cool to generate all the images for everything. 😄

Reply:
Yea once we get a cool .NET MAUI icon of .NET MAUI bot testing stuff we can switch to that one via Resizetizer